src/index.sh

```diff
@@ -71,7 +72,7 @@ exit_status=0
 csdiff --fixed "../dest-br-shellcheck.err" "../pr-br-shellcheck.err" > ../fixes.log
 
 if [ -s ../fixes.log ]; then
-  echo -e "✅ ${GREEN}Fixed bugs${NOCOLOR}"
+  echo -e "✅ ${GREEN}Fixed bugs:${NOCOLOR}"
```

Comment:
Hmm, there was `:` before but I removed it because it looks better without `:` IMHO. But I don't mind adding it back.

What do you think?